koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -475,7 +475,16 @@ def convert_csv_to_parquet(self, csv_filename):  # noqa: C901
             col_names = pd.read_csv(csv_filename, nrows=0, **kwargs).columns
             if self.ingress_reports:
                 REQUIRED_COLS = set(CSV_REQUIRED_COLUMNS.get(self._provider_type))
+                missing_cols = False
                 if not set(col_names).issuperset(REQUIRED_COLS):
+                    missing_cols = True
+                    if missing_cols and self._provider_type in {Provider.PROVIDER_AWS, Provider.PROVIDER_AWS_LOCAL}:
+                        REQUIRED_CUSTOM_COLS = set(CSV_REQUIRED_COLUMNS.get("AWS-custom"))
```

Comment:
A `KeyError` more clearly shows a potential problem here as well.
```suggestion
                        REQUIRED_CUSTOM_COLS = set(CSV_REQUIRED_COLUMNS["AWS-custom"])
```